seal_rookery/convert_images.py

```diff
@@ -124,13 +124,14 @@ def main(argv=None):
                         help='turn on verbose seal generation messages')
 
     args = parser.parse_args(argv)
-    changed, skipped = convert_images(
-        verbose=bool(args.v), forced=bool(args.f)
-    )
-
-    save_new_json()
+    try:
+        changed, skipped = convert_images(
+            verbose=bool(args.v), forced=bool(args.f)
+        )
+        save_new_json()
+    except Exception:
+        return 1
```

Comment:
My worry is that a non-zero exit code is really easy to miss, so if no exception and you miss it, you'll think things worked properly.